src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -350,37 +361,51 @@ void prepare_primitive_fusing::fuse_bias(program &p) {
             return node.as<fully_connected>().get_primitive()->input_size == 3;
         };
 
-        if (node->get_output_layout().is_dynamic())
-            continue;
-
-        cldnn::tensor::value_type out_features = node->get_output_layout().feature();
-        bool is_3d_fc = false;
 
-        // Change out_features value to proper dimension for 3D FC case
-        if (is_3d_fully_connected(node->get_dependency(0))) {
-            out_features = node->get_dependency(0).get_output_layout().spatial(1);
-            is_3d_fc = true;
-        } else if (is_3d_fully_connected(node->get_dependency(1))) {
-            out_features = node->get_dependency(1).get_output_layout().spatial(1);
-            is_3d_fc = true;
-        }
-
-        int bias_idx = -1;
-        for (size_t i = 0; i < eltw_node.get_dependencies().size(); i++) {
-            auto& dep = eltw_node.get_dependency(i);
-            if (dep.is_constant() &&
-                (dep.get_output_layout().feature() == out_features || is_3d_fc) &&
-                dep.get_output_layout().count() == static_cast<size_t>(out_features)) {
-                bias_idx = static_cast<int>(i);
-                break;
+        if (node->get_output_layout().is_dynamic()) {
+            auto broadcast_type = eltw_node.get_primitive()->broadcast_spec.m_type;
+            if (!eltw_node.get_dependency(non_const_dep_idx).is_type<fully_connected>())
+                continue;
+            if (broadcast_type != ov::op::AutoBroadcastType::NUMPY && broadcast_type != ov::op::AutoBroadcastType::NONE)
```

Comment:
Does not support PDPD case in fuse_bias?

Reply:
No. Because we cannot ensure that the feature size of the constant tensor is same as input for PDPD rule.